SDVModTest/ModConfig.cs

```diff
@@ -7,73 +7,103 @@
+            "0,0,0,0,0,0,0,0,0,0,0",
+            "0,0,0,0,0,0,0,0,0,0,0",
+            "0,0,0,0,0,0,0,0,0,0,0",
+            "0,0,0,0,0,0,0,0,0,0,0",
+            "0,0,0,0,1,1,1,0,0,0,0",
+            "0,0,0,0,1,0,1,0,0,0,0",
+            "0,0,0,0,1,1,1,0,0,0,0",
+            "0,0,0,0,0,0,0,0,0,0,0",
+            "0,0,0,0,0,0,0,0,0,0,0",
+            "0,0,0,0,0,0,0,0,0,0,0",
+            "0,0,0,0,0,0,0,0,0,0,0",
         };
 
-        public int[][] IridiumSprinkler { get; set; } = new int[][]
+        public string[] IridiumSprinkler { get; set; } = new string[]
         {
-            new int[] { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 },
-            new int[] { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 },
-            new int[] { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 },
-            new int[] { 0, 0, 0, 1, 1, 1, 1, 1, 0, 0, 0 },
-            new int[] { 0, 0, 0, 1, 1, 1, 1, 1, 0, 0, 0 },
-            new int[] { 0, 0, 0, 1, 1, 0, 1, 1, 0, 0, 0 },
-            new int[] { 0, 0, 0, 1, 1, 1, 1, 1, 0, 0, 0 },
-            new int[] { 0, 0, 0, 1, 1, 1, 1, 1, 0, 0, 0 },
-            new int[] { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 },
-            new int[] { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 },
-            new int[] { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }
+            "0,0,0,0,0,0,0,0,0,0,0",
+            "0,0,0,0,0,0,0,0,0,0,0",
+            "0,0,0,0,0,0,0,0,0,0,0",
+            "0,0,0,1,1,1,1,1,0,0,0",
+            "0,0,0,1,1,1,1,1,0,0,0",
+            "0,0,0,1,1,0,1,1,0,0,0",
+            "0,0,0,1,1,1,1,1,0,0,0",
+            "0,0,0,1,1,1,1,1,0,0,0",
+            "0,0,0,0,0,0,0,0,0,0,0",
+            "0,0,0,0,0,0,0,0,0,0,0",
+            "0,0,0,0,0,0,0,0,0,0,0",
         };
 
-        public int[][] Beehouse { get; set; } = new int[][]
+        public string[] Beehouse { get; set; } = new string[]
         {
-            new int[] { 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0 },
-            new int[] { 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0 },
-            new int[] { 0, 0, 0, 1, 1, 1, 1, 1, 0, 0, 0 },
-            new int[] { 0, 0, 1, 1, 1, 1, 1, 1, 1, 0, 0 },
-            new int[] { 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0 },
-            new int[] { 0, 1, 1, 1, 1, 0, 1, 1, 1, 1, 0 },
-            new int[] { 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0 },
-            new int[] { 0, 0, 1, 1, 1, 1, 1, 1, 1, 0, 0 },
-            new int[] { 0, 0, 0, 1, 1, 1, 1, 1, 0, 0, 0 },
-            new int[] { 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0 },
-            new int[] { 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0 }
+            "0,0,0,0,0,1,0,0,0,0,0",
+            "0,0,0,0,1,1,1,0,0,0,0",
+            "0,0,0,1,1,1,1,1,0,0,0",
+            "0,0,1,1,1,1,1,1,1,0,0",
+            "0,1,1,1,1,1,1,1,1,1,0",
+            "0,1,1,1,1,0,1,1,1,1,0",
+            "0,1,1,1,1,1,1,1,1,1,0",
+            "0,0,1,1,1,1,1,1,1,0,0",
+            "0,0,0,1,1,1,1,1,0,0,0",
+            "0,0,0,0,1,1,1,0,0,0,0",
+            "0,0,0,0,0,1,0,0,0,0,0"
         };
+
+        public int[][] getIntArray(string[] input)

```

Comment:
I honestly don't think the performance hit from having to convert strings is worth it.

Reply:
This change isnt my favorite but it was to create a human readable json file. The encoder places each number on a new line. If a user needs to modify the config directly it would be very difficult.